### PR TITLE
CI for Validation bundle (Part 1/2)

### PR DESCRIPTION
**Pull request by defectiveAi**
CI for Schemas package only.

### CODE REVIEW

File: .github/workflows/validation-bundle-publish.yml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+env:
+  SOLUTION_FILE_PATH: source/Validation/Validation.sln
+  BUILD_CONFIGURATION: Release
+  # Conditions
+  PUSH_PACKAGES: ${{ github.event_name != 'pull_request' }}
+  # Tool versions
+  DOTNET_VERSION: '5.0.202'
+  AZURITE_VERSION: '3.14.0'
+
```
Comment (dstsyst) [inline .github/workflows/validation-bundle-publish.yml:47]: As far as I can see AZURITE_VERSION is not used

File: .github/workflows/validation-bundle-publish.yml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            ${{ runner.os }}-nuget-
+      
+      - name: Restore NuGet packages
+        run: dotnet restore ${{ env.SOLUTION_FILE_PATH }}
+
+      - name: Build solution
+        run: dotnet build ${{ env.SOLUTION_FILE_PATH }} --no-restore --configuration ${{ env.BUILD_CONFIGURATION }}
+
+      - name: Run tests
+        run: |
```
Comment (dstsyst) [inline .github/workflows/validation-bundle-publish.yml:81]: If you only need to run one command, you can consider removing "|" and merge line 81 + 82 (kind of like the format of line 78)

File: .github/workflows/validation-bundle-publish.yml [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+          dotnet test ${{ env.SOLUTION_FILE_PATH }} --no-build --configuration ${{ env.BUILD_CONFIGURATION }} --verbosity normal
+
+      - name: Get version suffix property value
+        uses: ./.github/actions/get-version-suffix
```
Comment (dstsyst) [inline .github/workflows/validation-bundle-publish.yml:85]: Actually Thomas has implemented a better version of this in the ".github" repo; you can use it like:
`uses: Energinet-DataHub/.github/.github/actions/get-version-suffix@2.3.0`

File: .github/workflows/validation-bundle-publish.yml [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+        id: get_version_suffix
+      
+      - name: Pack Schemas project
+        run: dotnet pack ${{ env.PROJECT_PATH }} --configuration ${{ env.BUILD_CONFIGURATION }} --include-symbols -p:SymbolPackageFormat=snupkg -p:RepositoryCommit=${{ github.sha }} ${{ steps.get_version_suffix.outputs.version_suffix_property }}
+        env:
+          PROJECT_PATH: 'source/Validation/source/Schemas/Schemas.csproj'
+
+      - name: Check each NuGet package version does not exists (not looking at prereleases)
+        if: ${{ env.PUSH_PACKAGES == 'true' }}
+        uses: ./.github/actions/nuget-assert-release-version
```
Comment (dstsyst) [inline .github/workflows/validation-bundle-publish.yml:95]: Thomas has also moved this to ".github" so you can reference it like:
`uses: Energinet-DataHub/.github/.github/actions/nuget-assert-release-version@2.3.0`

**Review by dstsyst — COMMENTED** [review]
I have only reviewed the workflow

File: .github/workflows/validation-bundle-publish.yml (hunk 1 above)
Comment (defectiveAi) [reply .github/workflows/validation-bundle-publish.yml:46]: Copy/pasta error. Fixed.

File: .github/workflows/validation-bundle-publish.yml (hunk 3 above)
Comment (defectiveAi) [reply .github/workflows/validation-bundle-publish.yml]: Fixed.

File: .github/workflows/validation-bundle-publish.yml (hunk 4 above)
Comment (defectiveAi) [reply .github/workflows/validation-bundle-publish.yml]: Fixed.

**Review by michaelmajgaard — APPROVED** [review]

**Review by dstsyst — APPROVED** [review]

**Pull request merged**